kotlin/workflow-core/src/main/java/com/squareup/workflow/WorkflowAction.kt

```diff
@@ -43,7 +43,7 @@ interface WorkflowAction<StateT, out OutputT : Any> : (StateT) -> Pair<StateT, O
     /**
      * Returns a [WorkflowAction] that does nothing.
```

Comment:
Could elaborate a bit in the kdoc, something like: 

```
Returns a [WorkflowAction] that does nothing: no output is emitted, and `render` is called again with the same `state` as last time.
```

Reply:
Done.